x-pack/plugins/actions/server/builtin_action_types/server_log.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Joi from 'joi';
+
+import { ActionType, ActionTypeExecutorOptions } from '../types';
+
+const DEFAULT_TAGS = ['info', 'alerting'];
+
+const PARAMS_SCHEMA = Joi.object().keys({
+  message: Joi.string().required(),
+  tags: Joi.array()
+    .items(Joi.string())
+    .optional()
```

Comment:
per the [joi doc](https://github.com/hapijs/joi/blob/v16.0.0-rc2/API.md#anyoptional): 

> Used to annotate the schema for readability as all keys are optional by default.

I've tended to always add one of `required()` or `optional()` for "readability", but should probably go with the flow of whatever else is done in Kibana.  Did a quick check, didn't notice anywhere both were used, so we should probably get dir of the `.optional()` line.  Since one person was curious, likely others will be too :-)